feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/ui/item/SessionItem.kt

```diff
@@ -52,52 +51,22 @@ class SessionItem @AssistedInject constructor(
     override fun getLayout(): Int = R.layout.item_session
 
     override fun bind(viewBinding: ItemSessionBinding, position: Int) {
-        viewBinding.favorite.setOnClickListener {
-            sessionsViewModel
-                .favorite(session)
-        }
-        bindFavorite(session.isFavorited, viewBinding.favorite)
-        viewBinding.root.setOnClickListener {
-            viewBinding.root.findNavController()
-                .navigate(actionSessionToSessionDetail(session.id))
-        }
-        viewBinding.live.isVisible = session.isOnGoing
-        viewBinding.title.text = session.title.ja
-        viewBinding.room.text = session.room.name.getByLang(defaultLang())
-        viewBinding.survey.isEnabled = session.isFinished
-        imageRequestDisposables.clear()
-        viewBinding.speakers.bindSpeaker()
-    }
-
-    override fun bind(
-        viewBinding: ItemSessionBinding,
-        position: Int,
-        payloads: MutableList<Any>
-    ) {
-        if (payloads.isEmpty()) {
-            bind(viewBinding, position)
-        } else {
-            payloads.distinct().forEach { payload ->
```

Comment:
Thank you for investigate!!! Is this code not needed? 👀 
Sure, this code is to prevent the session from blinking.

Reply:
@eyedol 
Can you check this?? 🙏 
We are overriding `override fun getChangePayload` for SessionItem. But now we are not handling that. 👀

Reply:
> Thank you for investigating!!! Is this code not needed? 👀
> This code is to prevent the session from blinking.

Looks like it was removed from a merge conflict. Checking...

Reply:
@takahirom  fixed 🙇